autolink/postlink/gradleLinker.js

```diff
@@ -54,6 +57,22 @@ class GradleLinker {
     return contents;
   }
 
+  /**
+   * Check the current minSdkVersion specified and if it's lower than
+   * the required version, set it to the required version otherwise leave as it is.
+   */
+  _setMinSdkVersion(contents) {
+    var minSdkVersion = this._getMinSdkVersion(contents)
+    // If user entered minSdkVersion is lower than the default, set it to default.
+    if (minSdkVersion < DEFAULT_MIN_SDK_VERSION) {
+      debugn(`   Updating minSdkVersion to ${DEFAULT_MIN_SDK_VERSION}`)
+      return contents.replace(/minSdkVersion = \d*/, `minSdkVersion = ${DEFAULT_MIN_SDK_VERSION}`)
+    } 
+
+    debugn(`   Already specified minSdkVersion ${minSdkVersion}`)
+    return contents.replace(/minSdkVersion = \d*/, `minSdkVersion = ${minSdkVersion}`)
```

Comment:
lets account for white spaces around the equals sign
```suggestion
    return contents.replace(/minSdkVersion\s=\s\d*/, `minSdkVersion = ${minSdkVersion}`)
```

Reply:
Nice catch! Had to use `\s{0,}=\s{0,}` instead.